tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/ArchetypesTest.java

```diff
@@ -1,233 +0,0 @@
-
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import org.eclipse.reddeer.common.logging.Logger;
-import org.eclipse.reddeer.eclipse.jdt.debug.ui.jres.JREsPreferencePage;
-import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
-import org.eclipse.reddeer.junit.annotation.RequirementRestriction;
-import org.eclipse.reddeer.junit.requirement.matcher.RequirementMatcher;
-import org.eclipse.reddeer.junit.runner.RedDeerSuite;
-import org.eclipse.reddeer.requirements.jre.JRERequirement.JRE;
-import org.eclipse.reddeer.swt.api.TableItem;
-import org.eclipse.reddeer.swt.impl.table.DefaultTable;
-import org.eclipse.reddeer.swt.impl.toolbar.DefaultToolItem;
-import org.eclipse.reddeer.workbench.handler.WorkbenchShellHandler;
-import org.eclipse.reddeer.workbench.impl.editor.DefaultEditor;
-import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
-import org.eclipse.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
-import org.jboss.tools.central.reddeer.api.ExamplesOperator;
-import org.jboss.tools.central.reddeer.projects.ArchetypeProject;
-import org.jboss.tools.central.test.ui.reddeer.projects.AngularJSForge;
-import org.jboss.tools.central.test.ui.reddeer.projects.HTML5Project;
-import org.jboss.tools.central.test.ui.reddeer.projects.JavaEEWebProject;
-import org.json.simple.JSONArray;
-import org.json.simple.JSONObject;
-import org.json.simple.parser.JSONParser;
-import org.json.simple.parser.ParseException;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-/**
- * 
- * @author rhopp
- * @contributor jkopriva@redhat.com
- * @contributor vprusa@redhat.com
- */
-@JRE(cleanup=false)
-@RunWith(RedDeerSuite.class)
-public class ArchetypesTest {
-
-	@RequirementRestriction
-	public static Collection<RequirementMatcher> getRestrictionMatcher() {
-		return Arrays.asList(new RequirementMatcher(JRE.class, "version", "1.8"));
-	}
-	
-	@Retention(RetentionPolicy.RUNTIME)
-	@Target(ElementType.TYPE)
-	public @interface JRE {
-
-		/**
-		 * Cleanup.
-		 *
-		 * @return true, if successful
-		 */
-		boolean cleanup() default false;
-	}
-	
-	private static final String CENTRAL_LABEL = "Red Hat Central";
-	private static Map<org.jboss.tools.central.reddeer.projects.Project, List<String>> projectWarnings = new HashMap<org.jboss.tools.central.reddeer.projects.Project, List<String>>();
-	private static final Logger log = Logger.getLogger(ExamplesOperator.class);
-
-	private static final String BLACKLIST_ERRORS_REGEXES_FILE = "resources/blacklist-test-errors-regexes.json";
```

Comment:
I suppose you do not need this file anymore too.

Reply:
Well, yes, after checking with "grep -r" command, there is now no line of code that requires this file. Deleted.